runtime/lib/component/custom-config.js

```diff
@@ -0,0 +1,88 @@
+var logger = require('logger')('custom-config-runtime')
+var config = require('/etc/yoda/custom-config.json')
+var querystring = require('querystring')
+var safeParse = require('@yoda/util').json.safeParse
+
+class CustomConfig {
+  constructor (runtime) {
+    this.runtime = runtime
+  }
+
+  /**
+   * interce the configuration message for dnd-model etc
+   * @param msg
+   */
+  interce (msg) {
+    logger.info(`interce ${JSON.stringify(msg)}`)
+    if (msg.nightMode) {
+      if (typeof msg.nightMode === 'object') {
+        this.runtime.dndMode.setOption(msg.nightMode)
+        delete msg.nightMode
+      } else if (typeof msg.nightMode === 'string') {
+        var nightMode = safeParse(msg.nightMode)
+        if (nightMode) {
+
+          this.runtime.dndMode.setOption(nightMode)
+        }
+        delete msg.nightMode
+      }
+    }
+  }
+
+  /**
+   * Handling the configs from RokidApp, includes activation words, night mode, and etc..
+   * @param {string} message
+   */
+  onCustomConfig (message) {
+    var appendUrl = (pathname, params, stringify) => {
+      var obj
+      if (stringify) {
+        obj = {param: JSON.stringify(params)}
+      } else {
+        obj = params
+      }
+      return `yoda-skill://custom-config/${pathname}?${querystring.stringify(obj)}`
+    }
```

Comment:
Why is the arrow function inside here? it seems no any closure arguments inside it.

Reply:
And no need to be an arrow function, no `this` is used.